public/app/plugins/panel/table/TablePanel.tsx

```diff
@@ -145,7 +146,12 @@ function onChangeTableSelection(val: SelectableValue<number>, props: Props) {
 // placeholder function; assuming the values are already interpolated
 const replaceVars: InterpolateFunction = (value: string) => value;
 
-const getCellActions = (dataFrame: DataFrame, field: Field) => {
+const getCellActions = (
+  dataFrame: DataFrame,
+  field: Field,
+  valueRowIndex: number,
```

Comment:
should we just rename this to `rowIndex` everywhere? feels weird having two slightly different names from same thing. (yes we also have uPlot's `dataIdx`, but let's leave that one for now :sweat_smile:)